notebooks/example.livemd

```diff
@@ -187,7 +110,15 @@ defmodule ServerWeb.ExampleLive do
   use ServerNative, :live_view
 
   @impl true
-  def render(assigns), do: ~H""
+  def render(assigns),
+    do: ~H"""
+    <p phx-click="clicked">Hello, world!</p>
+    """
+
+  def handle_event("clicked", _params, socket) do
+    IO.inspect("CLICKED")
```

Comment:
did you mean to leave this here?

Reply:
Yes, this is part of the code that makes testing event handlers easier. It's inside of the code notebook.